lib/analytics-helper/index.js

```diff
@@ -96,6 +96,8 @@ export const addAnalyticsImpression = (item, dataLayer, impressionsTimestamp) =>
  * @returns {{event: string, preference_input: *, added_tags: *, displayed_tags: string}}
  */
 export const analyticsAddTagObject = (displayName, currentTags, context) => {
+  const initialTagIndex = currentTags.indexOf('Top inspiration');
```

Comment:
We should get the Tag `Top Inspiration` from the constant https://github.com/numo-labs/isearch-ui/blob/master/src/constants/default-tag.js